scripts/docs/to-product-docs

```diff
@@ -39,12 +39,12 @@ convert_relative_links() {
 
 # Changes .md extensions to .html
 convert_doc_extensions() {
-  sed -E -e '/(development|signalfx-agent.1|MANUAL).md/! s/\.md(#.*)?[[:>:]]/.html\1/g' -e'/(development|signalfx-agent.1).md/! s@\./docs/@./@g'
+  sed -E -e '/http:|https:|(development|signalfx-agent.1|MANUAL).md/! s/\.md(#.*)?[[:>:]]/.html\1/g' -e'/(development|signalfx-agent.1).md/! s@\./docs/@./@g'
 }
 
 reset_repo() {
   if ! git -C $PRODUCT_DOCS_REPO diff --exit-code; then
-    echo "Product docs repo has uncommitted changes, please deal with them first" >&2 
+    echo "Product docs repo has uncommitted changes, please deal with them first" >&2
```

Comment:
Hi - what does this mean? Is this a script to push directly to master? Or does it mean the specific branch you are trying to push to has uncommitted changes?

Reply:
I just trimmed the whitespace so shouldn't have a functional change. Apparently it's a piece of broken code though, pushed a change to fix it. @keitwb is reset_repo still valid?

Reply:
You have to set `MANAGE_GIT=yes` in the environment, but yes it should theoretically work.  It just wipes out any other changes in product-docs locally so you have a fresh copy.  We could also use the git worktree thing I was telling you about.